pxr/usd/usdGeom/generatedSchema.usda

```diff
@@ -4194,6 +4194,36 @@ class Camera "Camera" (
     However, it follows that if even one property is authored in the correct
     scene units, then they all must be.
 
+    Camera Exposure Model
+
+    UsdGeomCamera models exposure by a camera in terms of exposure time, ISO,
+    f-number, and exposure compensation, mirroring the controls on a real camera.
+    These parameters are provided by \\ref UsdGeomCamera::GetExposureTimeAttr(),
+    
+    and respectively. 
```

Comment:
This section of the generated schema is kinda messed up, and drops a few lines from the doxygen.  As @meshula noted, schema.usda has a lingering reference to GetFNumberAttr which doesn't exist; possibly that's causing this?